### ISSUE TITLE
Server has high lag/low TPS

### ISSUE THREAD

**Issue by cggentry115** [doctorray117/minecraft-ondemand#28]
Hi there, even after allocating 4 CPU units and 8GB of ram (which may not even be needed/helpful), I still seem to be having server issues. While my ping is the range of 100, it spikes up to 200 when breaking blocks, with TPS being between 12 and 13 on average. Any suggestions on how to improve performance?

**Comment by hojongs** [comment]
Did you configure the memory limit of the JVM in the server container? https://github.com/itzg/docker-minecraft-server#memory-limit
not only ECS task CPU/mem setting.

**Comment by fridaystreet** [comment]
yep I'm running 4 cpu, 8gb RAM, have the JVM set to 6gb and runs like a charm
